src/aarch64/neon/tests.rs

```diff
@@ -14,480 +13,15 @@ macro_rules! transmute {
 
 #[test] //FIXME: #[simd_test(enable = "neon")]
 fn test_vpaddq_u8() {
-    let a = i8x16::new(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
-    let b = i8x16::new(
+    let a = int8x16_t::new([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]);
+    let b = int8x16_t::new([
         17, 18, 19, 20, 20, 21, 22, 23, 24, 25, 26, 27, 29, 29, 30, 31,
-    );
```

Comment:
nice! I was thinking at some point we might want to add in similar tests as "smoke" tests. Will add an issue for that...